azure-cli/synapse/Publish/pipelines/templates/publish.synapse.parameters.ps1

```diff
@@ -0,0 +1,37 @@
+param (
+  [Parameter(Mandatory = $True)]
+  [ValidateSet("dev", "staging", "prod")]
+  [string]
+  $EnvironmentName,
+      
+  [Parameter(Mandatory = $True)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $SynapseWorkspaceParameterJsonPath,
+      
+  [Parameter(Mandatory = $True)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $OutputPath,
+
+  [Parameter(Mandatory = $False)]
+  [ValidateNotNullOrEmpty()]
+  [string]
+  $ParameterJsonPath
+)
+
+
+. "$PSScriptRoot\Initialize-SynapseParameters.ps1" # Change this to the correct location of the function
+
+$updateParams = @{ 
+  cowhdev = "cowh$EnvironmentName"
+  #curentvalue  = "replaceValue"
+  # ADD YOUR GENERAL PARAMETERS HERE..... 
+}
+  
+Initialize-SynapseParameters `
+  -EnvironmentName $EnvironmentName `
+  -OutputPath $OutputPath `
+  -SynapseWorkspaceParameterJsonPath $SynapseWorkspaceParameterJsonPath `
+  -ReplaceStringValues $updateParams
+# -ParameterJsonPath $ParameterJsonPath
```

Comment:
Since you're comitting the synapse.{dev,staging,prod}.parameters.json files, you might aswell send the uncomment -ParameterJsonPath and give it a real value